repository/myPageRepository.js

```diff
@@ -0,0 +1,180 @@
+const connection = require("../db/connection");
+const query = require("../db/query.json");
+
+async function selectMyPageSetting(user) {
+    let conn;
+    try {
+        conn = await connection();
+        const [[row]] = await conn.execute(query.myPage.select, [user.id]);
+
+        return row;
+    } catch(error) {
+        throw error;
+    } finally {
+        if(conn) conn.release();
+    }
+}
+
+async function updateMyPageSetting(body, user) {
+    let conn;
+    try {
+        conn = await connection();
+        const [row] = await conn.execute(query.myPage.update, [body.nickname, body.phone, user.id]);
+
+        return row;
+    } catch(error) {
+        throw error;
```

Comment:
```suggestion
        conn?.rollback();
        throw error;
```

메서드호출전에 물음표는 if(conn) conn.rollback() 과 같은 역할입니다.
conn이 null이 아니라면 해당 메서드를 호출합니다.
conn이 null이나 undefined라면 rollback메서드는 호출되지 않습니다.